.ado/jobs/playground.yml

```diff
@@ -84,19 +93,57 @@ jobs:
             /p:PreferredToolArchitecture=$(MSBuildPreferredToolArchitecture)
             /p:PlatformToolset=$(MSBuildPlatformToolset)
             $(BuildWinUI3Properties)
-            /p:BaseIntDir=$(BaseIntDir) 
+            /p:BaseIntDir=$(BaseIntDir)
+            /p:AppxPackageSigningEnabled=false
+        condition: eq('${{ parameters.BuildEnvironment }}', 'PullRequest')
+
+      - task: VSBuild@1
+        displayName: VSBuild - Playground CI
+        inputs:
+          solution: packages/playground/windows/$(SolutionFile)
+          vsVersion: $(MSBuildVersion) # Optional. Options: latest, 16.0, 15.0, 14.0, 12.0, 4.0
+          msbuildArchitecture: $(MSBuildArchitecture) # Optional. Options: x86, x64
+          platform: $(BuildPlatform) # Optional
+          configuration: $(BuildConfiguration) # Optional
+          clean: true # Optional
+          maximumCpuCount: false # Optional
+          restoreNugetPackages: false # Optional
+          msbuildArgs:
+            /p:PreferredToolArchitecture=$(MSBuildPreferredToolArchitecture)
+            /p:PlatformToolset=$(MSBuildPlatformToolset)
+            $(BuildWinUI3Properties)
+            /p:BaseIntDir=$(BaseIntDir)
+            /p:PackageCertificateKeyFile=$(Build.SourcesDirectory)\EncodedKey.pfx
+        condition: eq('${{ parameters.BuildEnvironment }}', 'Continuous')
+
+      - task: PowerShell@2
+        displayName: Remove Certificate
+        inputs:
+          targetType: 'inline'
+          script: |
+            $PfxPath = Join-Path -Path $(Build.SourcesDirectory) -ChildPath playground-Key.pfx
+            Write-Host $PfxPath
+            Remove-Item -path $PfxPath
+        condition: eq('${{ parameters.BuildEnvironment }}', 'Continuous')
 
       - powershell: $(Build.SourcesDirectory)/.ado/scripts/TestWACK.ps1 $(BuildPlatform) $(BuildConfiguration) $(Build.StagingDirectory)\WACK
         displayName: Test WACK
-        condition: and(succeeded(), eq(variables.TestWACK, true))
+        condition: and(succeeded(), eq(variables.TestWACK, true), eq('${{ parameters.BuildEnvironment }}', 'Continuous'))
 
       - task: PublishBuildArtifacts@1
         displayName: Upload WACK report
-        condition: and(succeededOrFailed(), eq(variables.TestWACK, true))
+        condition: and(succeededOrFailed(), eq(variables.TestWACK, true), eq('${{ parameters.BuildEnvironment }}', 'Continuous'))
         inputs:
           pathtoPublish: '$(Build.StagingDirectory)/WACK'
           artifactName: 'WACK report $(BuildPlatform) $(BuildConfiguration) ($(System.JobAttempt))'
 
+      - task: PublishBuildArtifacts@1
```

Comment:
This looks pretty useful, should we file an issue to forward port this to main branch if not there?

Reply:
It should be there, since this was a straight cherry-pick of relevant PRs.